internal/pkg/describe/service.go

```diff
@@ -108,24 +113,35 @@ func newServiceStackDescriber(opt NewServiceConfig, env string) (*serviceStackDe
 
 // Params returns the parameters of the service stack.
 func (d *serviceStackDescriber) Params() (map[string]string, error) {
+	if d.params != nil {
```

Comment:
Can we think of any scenario where caching these values will be problematic? is it at all possible that a stack update happens in between calls and causes issues or do you think we're safe?

Reply:
I was thinking about the same question but couldn't think of a scenario. Unless we want to use the same struct to get params and then wait and then get it again. Maybe for `status/show` if we want to support "refreshing periodically to get the latest update" like what `--follow` does for `logs`. However, I think we could add a field to disable this caching when we need it. What do you think?